internal/logger/logger.go

```diff
@@ -0,0 +1,86 @@
+package logger
+
+import (
+	"net/http"
+	"time"
+
+	"go.uber.org/zap"
+)
+
+// Log будет доступен всему коду как синглтон.
+// По умолчанию установлен no-op-логер, который не выводит никаких сообщений.
+var Log *zap.Logger = zap.NewNop()
```

Comment:
CHANGE REQUEST: please don't use global variables

Reply:
deleted the global variable